azerothcore/azerothcore-wotlk
# fix(Core/Player): Remove mount aura on a flight

## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7778
- Closes https://github.com/chromiecraft/chromiecraft/issues/1661

## SOURCE:


## Tests Performed:

- Tested
- Archlinux


## How to Test the Changes:


1 - Get on a gryphon, flying heading toward anywhere location
2 - While flying start RDF and search for a dungeon where you can mount inside of (zul' farrak)
3 - Once your are inside an instance, get on your mount
4 - Teletrasport outside using the eye button near the minimap (the rdf button)
5 - Once your gryphon arrives at destination You should not have a mount aura

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
